src/zcl_abapgit_repo.clas.abap

```diff
@@ -344,6 +350,12 @@ CLASS ZCL_ABAPGIT_REPO IMPLEMENTATION.
     APPEND INITIAL LINE TO rt_files ASSIGNING <ls_return>.
     <ls_return>-file = build_dotabapgit_file( ).
 
+    ls_apack_file = build_apack_manifest_file( ).
+    IF ls_apack_file IS NOT INITIAL.
+      APPEND INITIAL LINE TO rt_files ASSIGNING <ls_return>.
+      <ls_return>-file = build_apack_manifest_file( ).
```

Comment:
I am not sure why you build the file for the 2nd time here. See line 353.

Reply:
You're right. Doesn't make any sense and will be corrected in a minute...